packages/synapse-react-client/src/components/AccessRequirementList/RequirementItem/SelfSignAccessRequirementItem.test.tsx

```diff
@@ -14,7 +14,7 @@ import {
   mockToUAccessRequirement,
 } from '../../../mocks/accessRequirement/mockAccessRequirements'
 import userEvent from '@testing-library/user-event'
-import { MarkdownSynapseProps } from '../../Markdown'
+import { MarkdownSynapseProps } from '../../Markdown/MarkdownSynapse'
```

Comment:
Markdown/index.ts was removed, so imports had to be rewritten. See [this comment](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/1057/files#r1671242756) for more info.